Sources/CX10/xla_tensor_ops_wrapper_generated.cc.inc

```diff
@@ -2038,14 +2128,127 @@ class Tanh : public Node {
  private:
 };
 
+class TfOnehot : public Node {
```

Comment:
Where does the capitalization for these class name originate? It's a bit off.

Reply:
```
def snake_to_camel(name):
  return "".join(map(lambda x: x.capitalize(),name.split("_")))
```